doc/src/pair_hybrid.rst

```diff
@@ -260,7 +288,10 @@ the specific syntax, requirements and restrictions.
 ----------
 
 The potential energy contribution to the overall system due to an
-individual sub-style can be accessed and output via the :doc:`compute pair <compute_pair>` command.
+individual sub-style can be accessed and output via the :doc:`compute
```

Comment:
Confused by this unscaled comment.  Is that somehow different (for some pair styles?) than what the user would expect?

Reply:
I put this here to emphasize that the scaling only happens when when contributions are summed. People may get confused when the (plain) sum of the contributions from `compute pair` is different than the value of `epair`.